packages/fiori/src/themes/Bar.css

```diff
@@ -18,15 +18,15 @@
 }
 
 .ui5-bar-root .ui5-bar-startcontent-container {
-	padding-inline-start: 0.5rem;
+	padding-inline-start: 0.75rem;
 	display: flex;
 	flex-direction: row;
 	align-items: center;
 	justify-content: flex-start;
 }
 
 .ui5-bar-root .ui5-bar-content-container {
-	min-width: 30%;
+	min-width: calc(30% - 4rem); /* Remaining space after removing paddings and margins */
```

Comment:
Why 4rem? How did we calculate this? Can we replace this 4rem with a sum of 4-5 variables for each part of the calculation?

Reply:
The way I calculated the 4 rem was:
**2 x 0.75rem - // 1.5rem**
```
.ui5-bar-root .ui5-bar-startcontent-container {
	padding-inline-start: 0.75rem;
	
.ui5-bar-root .ui5-bar-endcontent-container {
	padding-inline-end: 0.75rem;
```

Following up -
**1.5rem + 1rem which comes from** - 

```
.ui5-bar-root .ui5-bar-midcontent-container {
	padding-left: 0.5rem;
	padding-right: 0.5rem;
```

**Current total sum - 2.5rem**

And now I finally add 1.5rem from here: // Which looking at it now is I think the case for the wrong calculation. Am I correct or the error is somewhere else ?

```
::slotted(*) {
	margin: 0 0.25rem; // for each slotted element in the start,mid or end we have 0.5rem - 0.25rem for each left and right
}
```
 **0.5rem x 3 = 1.5rem => 2.5rem + 1.5rem = 4rem;**

Reply:
Extract these numbers in variables and use calc() function.